packages/flutter/test/widgets/focus_traversal_test.dart

```diff
@@ -1874,6 +1875,113 @@ void main() {
       expect(controller.offset, equals(0.0));
     }, skip: isBrowser, variant: KeySimulatorTransitModeVariant.all()); // https://github.com/flutter/flutter/issues/35347
 
+    testWidgets('Focus traversal inside a horizontal scrollable applies ensure visible padding.', (WidgetTester tester) async {
+      tester.binding.focusManager.defaultScrollRevealPadding = const EdgeInsets.all(50.0);
```

Comment:
Can you also add a test that uses unique values for each edge to verify they are oriented properly in those cases?